vana/chain_manager.py

```diff
@@ -213,16 +213,16 @@ def send_transaction(self, function: ContractFunction, account: LocalAccount, va
         retry_count = 0
         while retry_count < max_retries:
             try:
-                # Estimate gas
+                # Estimate gas with 2x buffer
                 gas_limit = function.estimate_gas({
                     'from': account.address,
                     'value': self.web3.to_wei(value, 'ether')
                 }) * 2
 
-                # Get current gas price and increase it slightly for replacement transactions
-                gas_price = self.web3.eth.gas_price
-                if retry_count > 0:
-                    gas_price = int(gas_price * 1.1)  # Increase gas price by 10% on retries
+                # Start with a higher base gas price and increase aggressively on retries
+                base_gas_price = self.web3.eth.gas_price
+                gas_multiplier = 1.5 + (retry_count * 0.5)  # 1.5x, 2x, 2.5x for each retry
```

Comment:
Can we pass this value into the send_transaction function? With a default value of 1.5.

Reply:
Yes, good suggestion.